distributed/comm/tests/test_ws.py

```diff
@@ -155,6 +155,29 @@ async def test_http_and_comm_server(cleanup, dashboard, protocol, security, port
                 assert result == 11
 
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "protocol,security",
+    [
+        (
+            "ws://",
+            Security(extra_conn_args={"headers": {"Authorization": "Token abcd"}}),
+        ),
+        (
+            "wss://",
+            Security.temporary(
+                extra_conn_args={"headers": {"Authorization": "Token abcd"}}
+            ),
+        ),
+    ],
+)
+async def test_connection_made_with_extra_conn_args(cleanup, protocol, security):
+    async with Scheduler(protocol=protocol, security=security) as s:
+        connection_args = security.get_connection_args("worker")
+        comm = await connect(s.address, **connection_args)
+        assert comm.sock.request.headers.get("Authorization") == "Token abcd"
```

Comment:
There is a failing test in CI that is about some resource not being cleaned up.  It could be that this would be resolved by closing this comm

Maybe the following would work

```suggestion
        assert comm.sock.request.headers.get("Authorization") == "Token abcd"
        await comm.close()
```